javascript/app-framework/docs/report/app-framework.api.md

```diff
@@ -167,6 +167,11 @@ export function useDatasetMenuItems(selected: string | undefined, onClick: (tabl
 // @public
 export function useDataTableSource(pkg: DataTable | undefined): Maybe<TableContainer>;
 
+// Warning: (ae-missing-release-tag) "useHelpOnMount" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export function useHelpOnMount(key: string): void;
```

Comment:
I suspect there may be other help request types too, such as on particular control/element focus, not just component “mount”

Reply:
Sure, but you can use the API directly for that